webknossos/webknossos/dataset/dataset.py

```diff
@@ -964,7 +1088,11 @@ def get_or_create(
         return cls(dataset_path, voxel_size, name, exist_ok=True)
 
     def __repr__(self) -> str:
-        return repr("Dataset(%s)" % self.path)
+        return f"Dataset({repr(self.path)})"
+
+    def _ensure_writable(self) -> None:
+        if self._read_only:
```

Comment:
`View` exposes a `read_only` property. Perhaps that would also be a good idea for `Dataset` and `Layer`.

Reply:
Good idea, added it :+1: